### ISSUE TITLE
Unreadable "ps stderr" with custom JVM arguments

### ISSUE THREAD

**Issue by hyodocraft** [killpowa/Twickt-Launcher#1441]
**Describe the bug**
When I try to start an instance with Forge 43.1.3 (MC1.19.2), I get the following unreadable error.
(With the same setting and JVM arguments, when I start vanilla 1.19.2 it starts correctly and loads the save with no problems.)

> ps stderr: �R�}���h ���C�����������܂��B
> 
> (anonymous) @ actions.js:3159
> emit @ node:events:526
> addChunk @ node:internal/streams/readable:315
> readableAddChunk @ node:internal/streams/readable:289
> Readable.push @ node:internal/streams/readable:228
> onStreamRead @ node:internal/stream_base_commons:190
> 
> Process exited with code 1. Not too good..

**To Reproduce**
1. Set JVM Arguments in the General/Java settings as follows.
`-Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms5G -Xmx5G -server -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+UnlockDiagnosticVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M -XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1 -XX:UseAVX=2 -XX:+UseStringDeduplication -XX:+UseFastUnorderedTimeStamps -XX:+UseAES -XX:+UseAESIntrinsics -XX:UseSSE=4 -XX:+UseFMA -XX:AllocatePrefetchStyle=1 -XX:+UseLoopPredicate -XX:+RangeCheckElimination -XX:+EliminateLocks -XX:+DoEscapeAnalysis -XX:+UseCodeCacheFlushing -XX:+SegmentedCodeCache -XX:+UseFastJNIAccessors -XX:+OptimizeStringConcat -XX:+UseCompressedOops -XX:+UseThreadPriorities -XX:+OmitStackTraceInFastThrow -XX:+TrustFinalNonStaticFields -XX:ThreadPriorityPolicy=1 -XX:+UseInlineCaches -XX:+RewriteBytecodes -XX:+RewriteFrequentPairs -XX:+UseNUMA -XX:-DontCompileHugeMethods -XX:+UseFPUForSpilling -XX:+UseFastStosb -XX:+UseNewLongLShift -XX:+UseVectorCmov -XX:+UseXMMForArrayCopy -XX:+UseXmmI2D -XX:+UseXmmI2F -XX:+UseXmmLoadAndClearUpper -XX:+UseXmmRegToRegMoveAll -Dfile.encoding=UTF-8 -Xlog:async -Djava.security.egd=file:/dev/urandom -Dio.netty.maxDirectMemory=0`
2. Create an instance of Forge 43.1.3.
3. Launch the instance.

**Expected behavior**
The instance starts without problems.

**Possible solutions**

**Screenshots**
![GDLauncher_2022-08-29_01-32-27](https://user-images.githubusercontent.com/48426079/187084852-b26519cf-337a-4b10-9cd1-525f28b277fc.png)

**Operating System:**
- OS: Windows 10 Pro
- Version: 21H1
- Java Version: jdk-18.0.2.1+1 from [Adoptium](https://adoptium.net/)
- Minecraft Version: 1.19.2
- Mod Loader: Forge 43.1.3

**Additional context**
JVM Arguments taken from [etil-minecraft-flags](https://github.com/etil2jz/etil-minecraft-flags)
Then I deleted `-XX:-UseBiasedLocking` and `--add-modules jdk.incubator.vector` and changed `-XX:UseAVX=3` to `-XX:UseAVX=2` according to the console logs when I started vanilla 1.19.2.

Also, the same error occurs in other instances e.g. ATM7Sky

**Comment by Eskaan** [comment]
Which Launcher version do you have? (Settings/General at the bottom)?
Are the Java args related to the crash?

**Comment by hyodocraft** [comment]
> Which Launcher version do you have? (Settings/General at the bottom)?

The launcher version is v1.1.25.

> Are the Java args related to the crash?

Sorry for not isolating the issue and lack of searching logs.
Apparently the error happens regardless of the java args I have set up.

I guess this issue is the same as #1396. (1.19 forge is unable to start and error messages are unreadable.)
or #1359, but I don't know why the launcher did NOT say "The command line is too long.".

----

I've downloaded and installed latest version (v1.1.29). The issue no longer occurs even with that java args / Java18.
Thank you so much for the update!
